tests/integration/test_migrate.sh

```diff
@@ -25,7 +25,12 @@ if [ "$init" != "systemd" ];then
     exit 0
 fi
 
-dockerPid=$(ps -C docker -o pid=|xargs)
+if ! systemctl is-active docker >/dev/null; then
+     echo "Docker daemon is not running"
+     exit 1
```

Comment:
https://github.com/projectatomic/atomic/blob/master/tests/integration/test_migrate.sh#L5-L10

We can change the message if you like.